patches/net/minecraft/world/level/block/entity/AbstractFurnaceBlockEntity.java.patch

```diff
@@ -86,10 +86,10 @@
      }
  
 -    private static boolean canBurn(RegistryAccess p_266924_, @Nullable RecipeHolder<?> p_301107_, NonNullList<ItemStack> p_155007_, int p_155008_) {
```

Comment:
The patch is still removing a method, I don't see how this is any different tbh

Reply:
The mappings problem is not about the removed method, but rather the parameter indices. Apparently instance methods having the hidden this parameter messes with the method parchment uses to map parameter names.

That said, I could see benefit to keeping the vanilla method around and passing in null for the block entity for the sake of not removing a vanilla method at all.

Reply:
I could keep the vanilla method around, but given it is private and would add complexity to the patch regarding the logic for handling when null is passed to call `getResultItem` like vanilla does I am not sure it is worth it.

But yes it is mainly related to parameter indices, and as a nice side effect it does make the patch file cleaner to read as it is just an extra parameter at the end of the various lines rather than modifying the middle of the line, which should hopefully ever so slightly reduce maintenance burden of those patches

Reply:
IMO this is a parchment application problem rather than a NeoForge problem. The raw srg names make it easy to remap these parameters appropriately.